modules/cms/classes/ComponentBase.php

```diff
@@ -38,6 +39,11 @@ abstract class ComponentBase extends Extendable
      */
     public $isHidden = false;
 
+    /**
+     * @var int Determines if component should cache default partial in onRender hook. Uses minutes.
```

Comment:
This should be in seconds, that's what Laravel 5.9 is moving to so might as well do it here.